metrics/codahale.go

```diff
@@ -121,8 +121,7 @@ func (c *CodaHale) IncFloatCounterBy(key string, value float64) {
 }
 
 func (c *CodaHale) measureSince(key string, start time.Time) {
-	d := time.Since(start)
-	go c.updateTimer(key, d)
+	c.updateTimer(key, time.Since(start))
```

Comment:
I see both StandardRegistry.GetOrRegister and StandardTimer.Update use sync.Mutex.
Ideally we should benchmark if this change makes things worse or maybe leave it like it is now